### PR TITLE
AC Automated Test - LRAC-8449 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LRAC-8449

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsWebContent.testcase [hunk 1]
```diff
@@ -122,11 +122,100 @@ definition {
...
+
+            ACSegments.saveSegment();
+        }
+
+        task ("Go to page and get data for segmented individuals") {
+            ACUtils.launchDXP();
+
+            User.logoutAndLoginPG(
+                userLoginEmailAddress = "ac@liferay.com",
+                userLoginFullName = "ac ac");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsWebContent.testcase]: Hi @AfonsoHSantos, I think you could login that way `ACUtils.launchDXP(userEmailAddress = "ac@liferay.com");` and so avoid using the `User.logoutAndLoginPG` macro

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsWebContent.testcase [hunk 2]
```diff
@@ -122,11 +122,100 @@ definition {
...
+                    pageName = "AC Page",
+                    siteName = "Site Name");
+
+                Pause(locator1 = "5000");
+
+                User.logoutPG();
+
+                Navigator.gotoSitePage(
+                    pageName = "AC Page",
+                    siteName = "Site Name");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsWebContent.testcase]: Hi @AfonsoHSantos, Just a curiosity, does the anonymous user count as an unsegmented individual?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsWebContent.testcase (hunk 1 above)
Comment (AfonsoHSantos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsWebContent.testcase]: Fixed, thanks for the suggestion

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsWebContent.testcase (hunk 2 above)
Comment (AfonsoHSantos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsWebContent.testcase]: Hi @ViniLopes20 , I have tested this now and found that it is not necessary. thanks for the remark.

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Review by alexcandidos — COMMENTED** [review]
Hi @AfonsoHSantos , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**